src/controllers/posts/likes.js

```diff
@@ -0,0 +1,44 @@
+import dbClient from '../../utils/dbClient.js'
+import { sendDataResponse } from '../../utils/responses.js'
+
+export const likePost = async (req, res) => {
+  const { postId } = req.params
+  const { id } = req.user
+  try {
+    const likeOnPost = await dbClient.post.findUnique({
+      where: { id: parseInt(postId) }
+    })
+    if (!likeOnPost) {
+      return sendDataResponse(res, 404, { error: 'Post not found' })
+    }
+    await dbClient.postLike.create({
```

Comment:
I can't see an update for the prisma schema file in this PR - was that part of another PR?

Reply:
Yeah from this PR yesterday:
https://github.com/boolean-uk/team-dev-server/pull/58